tests/platform_tests/test_first_time_boot_password_change/conftest.py

```diff
@@ -0,0 +1,104 @@
+import pytest
+import logging
+import time
+import pexpect
+from tests.platform_tests.test_first_time_boot_password_change.manufacture import manufacture
+from tests.platform_tests.test_first_time_boot_password_change.default_consts import DefaultConsts
+
+
+def pytest_addoption(parser):
+    parser.addoption("--feature_enabled", action="store", default='False', help="set to True if the feature is enabled")
+
+
+class currentConfigurations:
+    '''
+    @summary: this class will act as a global database to save current configurations and changes the test made.
+    It will help us track the current state of the system,
+    and we will be used as part of cleanup fixtures.
+    '''
+    def __init__(self):
+        self.currentPassword = DefaultConsts.DEFAULT_PASSWORD  # initial password
+
+
+currentConfigurations = currentConfigurations()
```

Comment:
Instance name same as class name. It would be better to capitalize first letter of the class name.